core/store/src/trie/mod.rs

```diff
@@ -410,11 +410,12 @@ pub struct Trie {
     pub counter: TouchedNodesCounter,
 }
 
-/// Stores reference count change for some Trie key and value.
+/// Stores reference count change for some key-value pair in DB.
 #[derive(BorshSerialize, BorshDeserialize, Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
 pub struct TrieRefcountChange {
-    /// Hash of TrieKey.
-    key_hash: CryptoHash,
+    /// Hash of the value and part of the DB key.
+    /// Used for uniting with shard id to get actual DB key.
+    value_hash: CryptoHash,
```

Comment:
`value` is also ambiguous. This is actually the hash of a trie node right?

Reply:
Not necessarily. I mention [here](https://github.com/near/nearcore/blob/master/docs/trie.md#trie-1) that it can be either serialized RawTrieNodeWithSize or value corresponding to some TrieKey. Also fixed comment below, because it was also incorrect.

Actually, I ran into this ambiguity so much times that I even think about splitting `ColState` to `ColStateNode` and `ColStateRecord`. Do you know any cases which would make that impossible? 

Though, I don't feel we will ever have time for doing this. The simplest option could be renaming `value` to `node_or_trie_value`.

Reply:
> it can be either serialized RawTrieNodeWithSize or value

Ah yes that is what I meant. Somehow in my mind value is also a trie node...

Reply:
> The simplest option could be renaming value to node_or_trie_value.

Do you mean `trie_node_or_value_hash`?

Reply:
>Do you mean `trie_node_or_value_hash`?

Yes